app/models/cinema_hall.rb

```diff
@@ -0,0 +1,3 @@
+class CinemaHall < ApplicationRecord
```

Comment:
these files should have `# frozen_string_literal: true`. Have you run rubocop? You can install extension in vsc to see robocop's prompts immediately:)

Reply:
thanks for spotting Ania! I forgot to run it of course :D